src/index.ts

```diff
@@ -57,7 +58,6 @@ program
     try {
       const res: Process.RawPayload[] = await getAll(mapOptions(options))
       const { raw, active, disabled } = options
-
       let processes: Process.RawPayload[]
```

Comment:
I think there's some refactoring that can be done here to improve readability and testability. Basically from ln 61 to ln 84 we filter or manipulate `res` into something that we then pass to `dir`. My recommendation would be to refactor the logic of that manipulation into a pure helper function that we can then unit test. The code here would then just call `getAll`, transform the output, and call `dir` on that output. Does that make sense?

Reply:
👍